### PR TITLE
Added migration docs

### PR DESCRIPTION
**Pull request by garethahealy**
https://github.com/DozerMapper/dozer/issues/399

### CODE REVIEW

**Comment by garethahealy** [comment]
@orange-buffalo ; can you review please.

File: docs/asciidoc/migration/v6-to-v61.adoc [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+**To:**
+[source,xml,prettyprint]
+----
+<mappings xmlns="http://dozermapper.github.io/schema/bean-mapping"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://dozermapper.github.io/schema/bean-mapping http://dozermapper.github.io/schema/bean-mapping.xsd">
+----
+
+=== 2. ???
```
Comment (orange-buffalo) [inline docs/asciidoc/migration/v6-to-v61.adoc]: I believe Spring namespace update can be already added here.

**Comment by garethahealy** [comment]
The docs use gitbook, and gitbook uses a convention of a SUMMARY.adoc file to generate the left hand menu. I'd forgot to add the migration links (will do now).
- https://github.com/DozerMapper/dozer/blob/master/docs/asciidoc/SUMMARY.adoc

The gitbook is currently generated by the release.sh
- https://github.com/DozerMapper/dozer/blob/master/docs/release.sh

Which expects you've already checked out the gh-pages repo. The gh-pages links to the gitbook.
- https://dozermapper.github.io

Hope that makes sense.

**Comment by garethahealy** [comment]
@orange-buffalo ; updated as per feedback.

**Pull request merged**